interpretation/deepseismic_interpretation/azureml_pipelines/dev/kickoff_train_pipeline.py

```diff
@@ -0,0 +1,8 @@
+"""
+Create pipeline and kickoff run
+"""
+from deepseismic_interpretation.azureml_pipelines.train_pipeline import TrainPipeline
+
+orchestrator = TrainPipeline("interpretation/deepseismic_interpretation/azureml_pipelines/pipeline_config.json")
```

Comment:
Suggest we create a script which kicks off the pipeline and takes 2 arguments: one to pipeline_config.json and the other to Experiment name.

Reply:
Modified kickoff_train_pipeline.py to take in two optional arguments in this commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/6b28306bed98730b3dd8948f00654c41ebad9815